todo_app/app.py

```diff
@@ -65,15 +87,23 @@ def index():
     @check_user_role_is_writer
     def add_to_do_item():
         name = request.form.get('name')
+        app.logger.info(f'adding to do item - {name}')
```

Comment:
This is a really natural log to add, but is it a good idea? I can easily imagine scenarios where users add potentially sensitive information into their todos, and suddenly we have personal data littering our production logs. For example, we might have todos like:
* "Send <friend> an email about <matter>"
* "Update all my accounts that currently use Chelsea123 as the password"
* Visit doctor about <condition>